hare/hare.go

```diff
@@ -241,8 +255,9 @@ func (h *Hare) onTick(ctx context.Context, id types.LayerID) {
 	// retrieve set from orphan blocks
 	blocks, err := h.msh.LayerBlockIds(h.lastLayer)
 	if err != nil {
-		logger.With().Error("no blocks for consensus", log.Err(err))
-		return
+		// we have to try achieve consensus on empty set or fail, otherwise it will be the delayed consensus process forever
+		logger.With().Error("", log.Err(err))
+		//return
```

Comment:
Are you sure you don't want to return here?

Reply:
Not sure. But If there is no difference, why not?